front/ui/ui-charts/src/speedSpaceChart/components/SpeedSpaceChart.tsx

```diff
@@ -142,6 +144,23 @@ const SpeedSpaceChart = ({
       }}
       tabIndex={0}
     >
+      <div className="speed-space-slider-container">
+        <Slider
+          className="speed-space-slider"
+          width={ZOOM_CONFIG.SLIDER_WIDTH}
+          min={Math.log(ZOOM_CONFIG.MIN_RATIO) / Math.log(1.1)}
+          max={Math.log(ZOOM_CONFIG.MAX_RATIO) / Math.log(1.1)}
+          value={Math.log(store.ratioX) / Math.log(1.1)}
+          onChange={(e) => {
+            const value = Math.max(ZOOM_CONFIG.MIN_RATIO, Math.pow(1.1, Number(e.target.value)));
```

Comment:
clamp both sides after pow to keep ratioX in range.
```suggestion
            const value = clamp(Math.pow(1.1, Number(e.target.value)), ZOOM_CONFIG.MIN_RATIO, ZOOM_CONFIG.MAX_RATIO)
```

Reply:
Done 👍